integration-cli/docker_cli_rename_test.go

```diff
@@ -38,6 +38,27 @@ func (s *DockerSuite) TestRenameRunningContainer(c *check.C) {
 	c.Assert(name, checker.Equals, "/"+newName, check.Commentf("Failed to rename container %s", name))
 }
 
+func (s *DockerSuite) TestRenameRunningContainerAndReuse(c *check.C) {
+	testRequires(c, DaemonIsLinux)
+	out, _ := dockerCmd(c, "run", "--name", "first_name", "-d", "busybox", "top")
+	c.Assert(waitRun("first_name"), check.IsNil)
+
+	newName := "new_name" + stringid.GenerateNonCryptoID()
```

Comment:
All these come from the previous test and I just added the reuse case that this PR is addressing. Should we cleanup other cases to be consistent or leave this consistent with others.